Survey/Resources/Assets/Assets.xcassets/ic_background.imageset/Contents.json

```diff
@@ -0,0 +1,12 @@
+{
+  "images" : [
+    {
+      "filename" : "ic_background.pdf",
+      "idiom" : "universal"
+    }
+  ],
+  "info" : {
+    "author" : "xcode",
+    "version" : 1
+  }
+}
```

Comment:
For pdf files, We'd prefer you also enable preserves vector presentation, we don't have to care about scaling images when their size has been changed.

<img width="254" alt="Screen Shot 2022-07-26 at 10 19 51" src="https://user-images.githubusercontent.com/24598204/180916078-3d3a224a-2e9b-4f1c-b8b7-ada730a013ea.png">

Reply:
Thanks, I fixed at https://github.com/nkhanh44/ic-survey-ios/pull/7/commits/a4d7507b1a6826fa989873a520a7f7abd1e6f0bc